app/package.json

```diff
@@ -29,8 +29,14 @@
   },
   "proxy": "http://localhost:4000",
```

Comment:
Did you create this branch off of main ? It looks as though you have duplicate changes to review which shouldn't occur in a PR. PR should only have what changed in this branch specifically - but if you didn't create off of the right branch that you are attempting to merge into, it will have diffs from other places. I am seeing the same diffs in all of these PRs. To fix, create a PR to the branch you branched this one off of to only get the diffs.

Reply:
Good to know, I fixed it.